src/momento/aio/simple_cache_client.py

```diff
@@ -72,7 +88,7 @@ async def __aexit__(
         traceback: Optional[TracebackType],
     ) -> None:
         await self._control_client.close()
-        await self._data_client.close()
+        await self._get_next_client().close()
```

Comment:
We should close all the data clients, right?

Reply:
yipes!  thanks for catching this

Reply:
+1 changing the hard code client count to 2 would leak client instances here

Reply:
fixed in https://github.com/momentohq/client-sdk-python/pull/125/commits/0ce5f6d5a74b38e9578722638c4a6363caee5303